### ISSUE TITLE
Front-end: Button focus outline insufficient?

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#171]
Focused state:
<img width="94" alt="Screen Shot 2023-06-16 at 2 56 53 PM" src="https://github.com/equalizedigital/accessibility-checker/assets/13153456/9158a471-c9c5-4267-8b4d-6cd19f2ae04f">

Unfocused:
<img width="70" alt="Screen Shot 2023-06-16 at 2 57 34 PM" src="https://github.com/equalizedigital/accessibility-checker/assets/13153456/e566eed6-74da-4e6c-bd57-8f7834985b63">

Maybe it's just me, but I didn't think I could tab to the button at all, and I ended up installing it on a different site before I realized that I could tab to the button and was on it. Is that just me not paying enough attention? Or is there a better way to indicate focus on this for keyboard users? Maybe shifting it slightly or adjusting the opacity in addition to the outline?

Or do we think this is sufficient?

**Comment by SteveJonesDev** [comment]
Yeah, I think the focus state can be improved. What about this @amberhinds?

![Screen Shot 2023-06-16 at 4 58 51 PM](https://github.com/equalizedigital/accessibility-checker/assets/2895788/15994f42-d2ca-4fa5-9386-8e77e0bf6025)

**Comment by amberhinds** [comment]
Yes, that's way more obvious. Thanks.
